pkg/maps/ctmap/ipv4.go

```diff
@@ -0,0 +1,160 @@
+package ctmap
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/cilium/cilium/common"
+	"github.com/cilium/cilium/common/types"
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/u8proto"
+	"net"
+	"unsafe"
+)
+
+var (
+	// Service4Map is the map representing the bpf local conntrack map.
+	Service4Map = bpf.NewMap(MapName4+"_services",
+		bpf.MapTypeHash,
+		int(unsafe.Sizeof(CtKey4{})),
+		int(unsafe.Sizeof(CtEntry{})),
+		MapNumEntriesLocal)
+
+	// Service4GlobalMap is the map representing the bpf global conntrack map.
+	Service4GlobalMap = bpf.NewMap(MapName4Global+"services",
```

Comment:
The name of `Service4Map` is `MapName4+"_services"` this one doesn't have an underscore, on purpose?